api/room/schema_query.py

```diff
@@ -152,6 +164,29 @@ def resolve_all_rooms(self, info, **kwargs):
         response = PaginatedRooms(**kwargs)
         return response
 
+    @Auth.user_roles('Admin')
+    def resolve_all_remote_rooms(self, info):
+        service = Credentials().set_api_credentials()
+        page_token = None
+        remote_rooms = []
+        while True:
+            calendar_list = service.calendarList().list(
+                pageToken=page_token).execute()
+            for calendar_list_entry in calendar_list['items']:
```

Comment:
Consider using room_object instead of calendar_list_entry. Kindly remove the \ in line 176

Reply:
Thanks @Bonifase. This has been implemented.